File: 26. Remove Duplicates from Sorted Array/main.py
"""
Author: Amal Matty Antony
"""
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        prev, pos = nums[0], 0
        for i, val in enumerate(nums):
            if prev != val:
                prev = val
                pos += 1
                nums[pos] = val
        return pos+1

File: 26. Remove Duplicates from Sorted Array/main.cpp
// Solution By Vandit Jain (jainvandit99)
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int k = 0;
        if(nums.size() == 0) return k;
        for(int i=0;i<nums.size();i++){
            if(nums[i] != nums[k]){
                nums[++k] = nums[i];
            }
        }
        return k+1;
    }
};